stage0/02-firmware/01-packages

```diff
@@ -1,2 +0,0 @@
-raspberrypi-bootloader
-raspberrypi-kernel
```

Comment:
this makes total sense;
ideally we'd have some  adi-kuiper-bootloader and kernel packages;
but that's a loooong way away